addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -209,9 +209,9 @@ def _generate_efaktur_invoice(self, delimiter):
                     if tax.amount > 0:
                         tax_line += line.price_subtotal * (tax.amount / 100.0)
 
-                invoice_line_unit_price = line.price_unit
+                invoice_line_unit_price = line.tax_ids.compute_all(line.price_unit)['total_excluded']
```

Comment:
please add comments on why you can't take directly `line.price_unit` and `invoice_line_unit_price * line.quantity`